recipe/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "nvcc" %}
+{% set number = 1 %}
+
+package:
+  name: {{ name }}
+  version: {{ cudatoolkit | replace(".*", "") }}
+
+build:
+  number: {{ number }}
+
+outputs:
```

Comment:
This seems setup a little different from the rest of our recipes. If there is only going to be a single output, we can probably avoid the `outputs` section and just move everything to the top level of the meta.yaml file.

Reply:
I agree. This is based on the one from conda-forge. I'll rework it to look a bit more standard.

Reply:
I think the package name can contain the platform - `name: {{ name }}_{{ target_platform }}`.

Reply:
I didn't include the `target_platform` part because this will build as an arch specific package already. We could move this to `noarch` I suppose, in which case we would want to add the platform to the name. We could then need to specify it that way in other feedstocks, like:

```
 requirements:
   - nvcc_linux-ppc64le    [ppc64le]
   - nvcc_linux-x86_64     [x86_64]
```

Which is not necessarily a good trade off, IMO.

Reply:
That would be handled by using the `{{ compiler("cuda") }}` way of specifying things that @cdeepali mentioned in the NCCL PR. However, I don't see what the benefit would be to adding the `target_platform` to the name, besides making it look like other build tools.

Reply:
Yes, like for `{{ compiler('c') }}` we have `gcc_linux-ppc64le` and `gcc_linux-64`, it would just look like other build tools.

Reply:
The meta.yaml layout here is actually correct now. When using the `compiler_cuda` method, we want the `{{ target_platform }}` string to be appended to the package name (this is required by the `{{ compiler }}` macros, but we still want to refer to `nvcc` in other places like the env files and the `conda_build_config.yaml`

So I'm going to resolve this topic.